### PR TITLE
Added API to get supported router features.

### LINKED ISSUE #284: Router should expose supported roles
We are using a different websocket server than gorilla for our wamp router. It would be ideal if the router could export an API so that we can query what dealer/broker roles it supports.

### PR DESCRIPTION
**Pull request by KSDaemon**
Added router method for getting currently supported features.

Closes #284

### What kind of change does this PR introduce?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:

- [x] My code follows the code style of this project.
- [x] I have added tests to cover my changes.
- [x] Overall test coverage is not decreased.
- [x] All new and existing tests passed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: router/router.go [hunk 1]
```diff
@@ -312,6 +317,16 @@ func (r *router) AddRealm(config *RealmConfig) error {
 	return err
 }
 
+// GetRouterFeatures exposes WAMP features current version provides.
+func (r *router) GetRouterFeatures() *wamp.Dict {
```
Comment (om26er) [inline router/router.go:321]: I think this should just be called `RouterFeatures`

Comment (KSDaemon) [reply router/router.go]: Yeah, renamed to be more go idiomatic

**Comment by KSDaemon** [comment]
@gammazero what do you think? Please have a look when you have time. This one is pretty small.

**Comment by KSDaemon** [comment]
Well, merging... As this is pretty small change + test covered/passed.

**Pull request merged**